scripts/start.sh

```diff
@@ -17,7 +21,16 @@ if [ "$NODE_ENV" != "production" ]; then
      [ ! -z "$TERM_PROGRAM_VERSION" ] && \
      [ $(version $TERM_PROGRAM_VERSION) -ge $(version "1.22.2") ]
   then
-    OPT+=(--inspect-brk)
+    # Sometimes we have force and metaphysics running at the same time.
+    # in that case we don't want
+    if (node --inspect-brk -e 'console.log()') &> /dev/null; then
```

Comment:
So what happens when there’s no other debugger server running? VSCode would connect to this newly spawned process, which would continue the halted process, and then immediately finish?

Reply:
yeah that's what it was doing. I just changed it to probe port 9229 with `nc`, since that's faster and avoids the little blip of orange in vscode if you have autoattach on.

Reply:
Nice